ntp-daemon/src/system.rs

```diff
@@ -212,6 +198,16 @@ impl<C: NtpClock, T: Wait> System<C, T> {
         )
     }
 
+    fn add_spawner(&mut self, spawner: impl Spawner + Send + Sync + 'static) -> SpawnerId {
+        let (notify_tx, notify_rx) = mpsc::channel(Self::MESSAGE_BUFFER_SIZE);
+        let id = spawner.get_id();
+        let spawner_data = SystemSpawnerData { id, notify_tx };
+        info!(id=?spawner_data.id, addr=spawner.get_addr_description(), "Running spawner");
```

Comment:
would it be clear here what kind of spawner this is? You could probably get a type name with the functions from `std::any` if that is helpful

Reply:
I've added a get_description() function to the trait to help with this and will display that as well to make sure that we know what kind of spawner we are talking about in the logs